### PR TITLE
Updated readme.md : seprated AutoGen and EcoOptGen also removed bibtex

### PR DESCRIPTION
**Pull request by priyansh4320**
new changes :

seperated AutoGen and EcoOptGen also removed bibtex

## Citation  
[AutoGen](https://arxiv.org/abs/2308.08155). 
``` 
@inproceedings{wu2023autogen,
      title={AutoGen: Enabling Next-Gen LLM Applications via Multi-Agent Conversation Framework},
      author={Qingyun Wu and Gagan Bansal and Jieyu Zhang and Yiran Wu and Shaokun Zhang and Erkang Zhu and Beibin Li and Li Jiang and Xiaoyun Zhang and Chi Wang},
      year={2023},
      eprint={2308.08155},
      archivePrefix={arXiv},
      primaryClass={cs.AI}
}
```

[EcoOptiGen](https://arxiv.org/abs/2303.04673).
```
@inproceedings{wang2023EcoOptiGen,
    title={Cost-Effective Hyperparameter Optimization for Large Language Model Generation Inference},
    author={Chi Wang and Susan Xueqing Liu and Ahmed H. Awadallah},
    year={2023},
    booktitle={AutoML'23},
}
```

 [MathChat](https://arxiv.org/abs/2306.01337). 

```
@inproceedings{wu2023empirical,
    title={An Empirical Study on Challenging Math Problem Solving with GPT-4},
    author={Yiran Wu and Feiran Jia and Shaokun Zhang and Hangyu Li and Erkang Zhu and Yue Wang and Yin Tat Lee and Richard Peng and Qingyun Wu and Chi Wang},
    year={2023},
    booktitle={ArXiv preprint arXiv:2306.01337},
}
```

### CODE REVIEW

**Comment by priyansh4320** [comment]
@microsoft-github-policy-service agree

**Comment by priyansh4320** [comment]
Please review @JieyuZ2

File: README.md [hunk 1]
```diff
@@ -145,3 +145,41 @@ Privacy information can be found at https://privacy.microsoft.com/en-us/
 
 Microsoft and any contributors reserve all other rights, whether under their respective copyrights, patents,
 or trademarks, whether by implication, estoppel or otherwise.
+
+
+## Citation  
+[AutoGen](https://arxiv.org/abs/2308.08155).
```
Comment (JieyuZ2) [inline README.md:151]: better separate these two papers

File: README.md [hunk 2]
```diff
@@ -145,3 +145,41 @@ Privacy information can be found at https://privacy.microsoft.com/en-us/
 
 Microsoft and any contributors reserve all other rights, whether under their respective copyrights, patents,
 or trademarks, whether by implication, estoppel or otherwise.
+
+
+## Citation  
+[AutoGen](https://arxiv.org/abs/2308.08155). 
+AND  [EcoOptiGen](https://arxiv.org/abs/2303.04673).
+``` 
+bibtex
```
Comment (JieyuZ2) [inline README.md:154]: also remove the 'bibtex' word, which is not necessary

Comment (priyansh4320) [reply README.md]: @JieyuZ2  okay done with changes :: review

**Review by JieyuZ2 — APPROVED** [review]

**Review by JieyuZ2 — APPROVED** [review]

**Pull request merged**